src/core/Password.cpp

```diff
@@ -40,8 +41,9 @@ std::shared_ptr<CPassword::SVerificationResult> CPassword::verify(const std::str
         auto auth = [&](std::string auth) -> bool {
             const pam_conv localConv = {conv, (void*)pass.c_str()};
             pam_handle_t*  handle    = NULL;
+            struct passwd *passwd = getpwuid(getuid());
```

Comment:
I'd rather drop the `struct` (this is C++) and just name the variable something like uidPassword

Reply:
I replaced `struct` with `auto` keyword with the suggested variable name.